ra/ra.go

```diff
@@ -1118,14 +1122,6 @@ func (ra *RegistrationAuthorityImpl) issueCertificateInner(
 		return emptyCert, berrors.MalformedError("certificate public key must be different than account key")
 	}
 
-	// Check rate limits before checking authorizations. If someone is unable to
-	// issue a cert due to rate limiting, we don't want to tell them to go get the
-	// necessary authorizations, only to later fail the rate limit check.
-	err = ra.checkLimits(ctx, names, account.ID)
-	if err != nil {
-		return emptyCert, err
-	}
-
```

Comment:
I think I need more convincing that we don't want to check rate limits in the RA at the time of certificate issuance for both ACME v1 and ACME v2. In this branch it will only happen at issuance time for ACME v1 issuance via NewCertificate. I'm not positive that there aren't rate limiting loop holes/pitfalls introduced when you don't enforce certificate rate limits at the time an order is finalized. Can you help me feel more confident this is a safe change?

Reply:
☝️

Reply:
Yep, I'll file a separate issue to talk about. Given that this is a "deprecate flags" PR, I'm going to just revert this part of it for now.

Reply:
Sounds good, thanks!